### DISCUSSION TITLE
Analysis with limited sorts of 10X BCR output

### DISCUSSION THREAD

**Discussion by jaesoonpark** [tuonglab/dandelion#254] [Q&A] [answered]
Hello. 

I am curious whether I can utilize the visualization tools having limited sorts of 10X BCR outputs.

The output currently available is as followed:

(1) BCR_filtered_contig_annotations.csv

(2-1) barcodes.tsv.gz
(2-2) features.tsv.gz
(2-3) matrix.mtx.gz

Hope I can get some insight about how to utilize the tool. 
Thanks for your help in advance.

**Comment by zktuong** [comment, marked as answer]
Hi @jaesoonpark, sorry for the delayed response - didn't realised i don't get ping when a question is raised here .

Anyway, yes! Dandelion has a `ddl.read_10x_vdj` function that will just take your first file (BCR_filtered_contig_annotations.csv).

You can read your single-cell object (number 2s) as per scanpy's tutorials, and then follow the rest of the tutorials for the various visualisations
